USca/USca_AlarmDisplay/AlarmAlerts.xaml

```diff
@@ -0,0 +1,35 @@
+﻿<Window
+    x:Class="USca_AlarmDisplay.AlarmAlerts"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:local="clr-namespace:USca_AlarmDisplay"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    Title="Alam alerts"
+    Width="800"
+    Height="450"
+    DataContext="{Binding RelativeSource={RelativeSource Self}}"
+    mc:Ignorable="d">
+    <Window.Resources>
+        <local:AlarmLogDTOToCustomStringConverter x:Key="logConverter" />
+    </Window.Resources>
+    <Grid Margin="10">
+        <Grid.RowDefinitions>
+            <RowDefinition />
+            <RowDefinition Height="auto" />
+        </Grid.RowDefinitions>
+        <ListBox Grid.Row="0" ItemsSource="{Binding AlarmLogs}">
+            <ListBox.ItemTemplate>
+                <DataTemplate>
+                    <TextBlock Text="{Binding {}, Converter={StaticResource logConverter}}" />
```

Comment:
This is low priority, but how about coloring based on priority? If you want to do this, you could make an issue for later, since it's not required of us to do.

Reply:
Will be done in subsequent PR